src/tsconfig.json

```diff
@@ -3,7 +3,9 @@
 	"compilerOptions": {
 		"rootDir": "./",
 		"outDir": "../dist",
-		"composite": true
+		"composite": true,
+		"preserveConstEnums": true,
```

Comment:
If merging #4 into here doesn't auto fix it then this property should be removed as it's already set in @sapphire/ts-config